decentraland/unity-explorer
# fix: double click equip wearables & emotes

Fixes #526 

# Test steps

1. Open the backpack
2. Double click on wearables from the grid
3. Observe that the item is equipped
4. Click on the equip button
5. Observe that the item is equipped
6. Go to emotes tab
7. Double click on emotes from the grid
8. Observe that the item is equipped
9. Click on the equip button
10. Observe that the item is equipped
